greenberry/utils.py

```diff
@@ -23,9 +23,9 @@ def eval(str):         #Function to evaluate postfix expression
         i=i+1
     print(S[0])
 
-def InfixCon():
```

Comment:
merging but make sure you follow pep8 at least like infix_con, exp = string instead of exp=string  etc. thanks for PR